plugin/kubernetes/ns.go

```diff
@@ -82,9 +88,10 @@ func (k *Kubernetes) nsAddrs(external bool, zone string) []dns.RR {
 		rr.Hdr.Name = svcNames[i]
 		rr.A = ip
 		rrs[i] = rr
+		i++
 	}
 
-	return rrs
+	return rrs[0:i]
```

Comment:
this can be shorted to `rrs[:i]` but I don't think we need this because uniqueness is done else where.